helm/bloom-vault/values.yaml

```diff
@@ -0,0 +1,104 @@
+# Default values for bloom-vault.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+#Global
+image:
+  tag: "latest"
+
+# Bloom Vault Server
+# Default values for bloom-vault server.
+server:
+
+  name: server
+  replicaCount: 2
+
+  image:
+    repository: hellobloom/bloom-vault
+    pullPolicy: Always
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+
+  imagePullSecrets:
+    - name: myregistrykey
+  nameOverride: ""
+  fullnameOverride: ""
+
+  envSecret: bloom-vault-env-secret
+  postgresSecret: bloom-vault-postgres-secret
+  postgresSecretMountPath: "/var/run/secrets"
+
+  containerCommand: "docker-entrypoint.sh"
+  containerArgs: 
+    - "/bin/bash"
+    - "-c"
+    - "bin/start.sh"
+
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  podAnnotations: {}
+
+  podSecurityContext: {}
+    # fsGroup: 2000
+
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  service:
+    type: NodePort
+    port: 3001
+    targetPort: 3001
+
+  ingress:
+    enabled: false
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: chart-example.local
+        paths: []
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  autoscaling:
+    enabled: false
```

Comment:
Should this be enabled and if so should the maxReplicas be set to much less?

Reply:
enabled with max of 5